tests/TestCase.php

```diff
@@ -16,7 +16,11 @@ public function setUp()
     {
         parent::setUp();
 
-        $this->loadLaravelMigrations();
+        if (app()->version() < 5.4) {
```

Comment:
We no longer support <5.4, so you can actually remove the conditional code entirely.  How was this "breaking"?

Reply:
@fletch3555 The `loadLaravelMigrations` method didn't exists.

Reply:
Maybe something with `orchestra/testbench-browser-kit`

Reply:
This would be my guess: https://github.com/orchestral/testbench-core/blob/b6ecfd57c72c0dff3288811ed4b8c68ffbd75afc/src/Concerns/WithLaravelMigrations.php#L16

Reply:
I found out that the `WithLaravelMigrations` trait was added to `v3.4.1` (https://github.com/orchestral/testbench-browser-kit/blob/v3.4.1/src/TestCase.php).

`v3.4.0`: https://github.com/orchestral/testbench-browser-kit/blob/v3.4.0/src/TestCase.php